src/pages/sighting/encounters/Encounters.jsx

```diff
@@ -219,11 +229,13 @@ export default function Encounters({
         ];
 
         const identifyButtonActions = [
-          {
-            id: 'view-id-results',
-            labelId: 'VIEW_IDENTIFICATION_RESULTS',
-            href: `/match-results/${sightingId}`,
-          },
+          isIdReady
+            ? {
+                id: 'view-id-results',
+                labelId: 'VIEW_IDENTIFICATION_RESULTS',
+                href: `/match-results/${sightingId}`,
+              }
+            : {},
```

Comment:
we shouldn't pass an empty object here, just omit the object if ~isIdReady

Reply:
@brmscheiner what do you think of this solution: https://github.com/WildMeOrg/codex-frontend/pull/390/commits/38faa05999e3185f0946071c8f506b860292b130?

I was also considering just splicing the object into an already-existing array if isIdReady, but that seemed less elegant.

Did you have another solution in mind?